hack/dind-cluster.sh

```diff
@@ -136,7 +130,10 @@ function start() {
   fi
 
   if [[ -n "${local_registry}" ]]; then
-    add-registry-to-runtime "${registry_ip}:${REGISTRY_PORT}" "${container_runtime}"
+    local registry_ip=$(get-registry-ip)
+    ensure-local-registry "${origin_root}" "${config_root}" "${registry_ip}" "${local_registry_images[@]}"
```

Comment:
`local_registry_images` is a string, and even if it were an array, `ensure-local-registry` is expecting only one positional parameter to represent the list of images; why use `[@]`?